### ISSUE TITLE
Doesn't seem to work with knex@2.0.0

### ISSUE THREAD

**Issue by cerinoligutom** [olavim/objection-cursor#35]
![image](https://user-images.githubusercontent.com/6721822/169815107-d246eeb7-b11b-4fb4-9694-37ac1caf56e1.png)

Have a `users` table with a `createdAt` field and I'm sorting using that field. Code:

```js
await UserModel.query().orderBy('asc', 'createdAt').limit(10).nextCursorPage(myCursor);
```

Code works fine on `knex@0.95.14` though. I think v1 is also broken as well since I've had a similar issue on one of my repo https://github.com/cerino-ligutom/GraphQL-Starter/issues/27#issuecomment-1065919299 where downgrading to `0.95.14` fixed the issue. I guess knex changed an underlying API that broke this plugin?

**Comment by olavim** [comment]
Thanks for reporting. I'll try to get this fixed ASAP.

**Comment by olavim** [comment]
What knex and objection versions are you using? And what database? I tested the following combinations with sqlite:

objection|knex|status
----------|-----|----------
`2.2.18`|`knex@0.95.14`| ✅ 
`2.2.18`|`knex@1.0.7`| ❌ 
`2.2.18`|`knex@2.0.0`|  ❌ 
`3.0.1`|`knex@0.95.14`| ✅ 
`3.0.1`|`knex@1.0.7`| ❌ 
`3.0.1`|`knex@2.0.0`|  ✅ 

Note: knex@1.0.7 failed because it depends on an sqlite3 package that no longer exists.

**Comment by cerinoligutom** [comment]
`knex@2.0.0`, `objection@3.0.1`, postgres:13

**Comment by olavim** [comment]
Can you check if version [1.2.6-alpha.0](https://www.npmjs.com/package/objection-cursor/v/1.2.6-alpha.0) fixes the issue for you?

**Comment by cerinoligutom** [comment]
Hey @olavim, works great! I don't get the error anymore 🎉

**Comment by olavim** [comment]
Great! Fixed in [v1.2.6](https://www.npmjs.com/package/objection-cursor/v/1.2.6).

**Comment by cerinoligutom** [comment]
Thank you for the quick fix! 🚀
